lib/iris/__init__.py

```diff
@@ -303,7 +315,312 @@ def _load_collection(uris, constraints=None, callback=None):
     return result
 
 
-def load(uris, constraints=None, callback=None):
+class LoadPolicy(threading.local):
+    """A container for loading strategy options.
+
+    Controls merge/concatenate usage, and the handling of cases where multiple reference
+    fields merge to define an additional dimension (e.g. a time-varying orography).
```

Comment:
When you say "reference fields" here, is this talking about a reference field *for a derived coordinate*? If so, it would be worth making that more explicit.